tower-web/src/app/modules/main/component/tasks-table/tasks-table.component.html

```diff
@@ -55,3 +55,16 @@
     </div>
   </div>
 </div>
+
+<div class="modal fade bd-example-modal-lg" id="exampleModalLong" tabindex="-1" role="dialog"
+     aria-labelledby="exampleModalLongTitle" aria-hidden="true">
+  <wt-task-details
+    [workflowId]="workflowId"
+    [taskId]="taskId"
+    [exitCode]="exitCode"
+    [env]="env"
+    [resTime]="resTime"
+    [resRequest]="resRequest"
+    [resUsed]="resUsed"
+  ></wt-task-details>
```

Comment:
Hi, `task-detail` component should take as input only the `taskId` and `workflowId`. The content should be created in the dialog component.